pq-crypto/pq_kem_params_size.h

```diff
@@ -0,0 +1,23 @@
+// -----------------------------------------------------------------------------
```

Comment:
Can this file be deleted? Probably we can only use the P434_api.h?

Reply:
Yes right now we don't need this file, but when we add another PQ KEM we'll need it because all the KEMs define the same macros with the same names so there would be a collision on the names if we don't define them separately in this file.